src/components/Changelog/IncidentHistoryItemTypeContent.tsx

```diff
@@ -0,0 +1,138 @@
+import { FaComment } from "react-icons/fa";
+import {
+  IncidentHistory,
+  IncidentHistoryType
+} from "../../api/services/IncidentsHistory";
+import { hypothesisStatusIconMap } from "../../constants/hypothesisStatusOptions";
+import { statusItems } from "../Incidents/data";
+
+type IncidentHistoryItemTypeContentProps = {
+  incidentHistory: IncidentHistory;
+};
+
+const IncidentHistoryTypeToComponentMap = new Map<
+  `${IncidentHistoryType}`,
+  React.FC<IncidentHistoryItemTypeContentProps>
+>([
+  [
+    "incident.created",
+    ({ incidentHistory }) => {
+      const incident = incidentHistory.incident;
+
+      return (
+        <div className="normal-case">
+          Created {incident?.description.toLowerCase()} incident
+        </div>
+      );
+    }
+  ],
+  [
+    "incident_status.updated",
```

Comment:
It looks like we have a dup of incident status updated ?

Reply:
Yeah, there is dup, which one should we go with?